bootstrap/kubeadm/api/v1beta1/kubeadm_types.go

```diff
@@ -53,6 +53,22 @@ type InitConfiguration struct {
 	// fails you may set the desired value here.
 	// +optional
 	LocalAPIEndpoint APIEndpoint `json:"localAPIEndpoint,omitempty"`
+
+	// CertificateKey sets the key with which certificates and keys are encrypted prior to being uploaded in
+	// a secret in the cluster during the uploadcerts init phase.
+	// +optional
+	CertificateKey string `json:"certificateKey,omitempty"`
+
+	// SkipPhases is a list of phases to skip during command execution.
+	// The list of phases can be obtained with the "kubeadm init --help" command.
+	// The flag "--skip-phases" takes precedence over this field.
+	// +optional
+	SkipPhases []string `json:"skipPhases,omitempty"`
+
+	// Patches contains options related to applying patches to components deployed by kubeadm during
+	// "kubeadm init".
```

Comment:
Also in this case min Kubernetes version should be documented
(same for join configuration)

Reply:
@asifdxtreme I think we should mention the Kubernetes version here too (like in the JoinConfiguration below)